src/nupic/algorithms/KNNClassifier.py

```diff
@@ -985,11 +1013,23 @@ def _getDistances(self, inputPattern, partitionId=None):
 
 
   def finishLearning(self):
+    """
+    TODO: Somebody help me write this.
+
+    :return:
+    """
```

Comment:
@numenta/nupic-reviewers I need help writing this. You can simply respond in a comment with a function description and I'll do the rest.